README.md

```diff
@@ -55,122 +91,67 @@ var i18n = require('hmpo-i18n');
 i18n.middleware(app);
 
 app.use(function (req, res, next) {
+    // load language from querystring parameter - e.g. ?lang=en
+    req.setLanguage(req.query.lang);
+    
     // a translate method is now available on the request
     // this will translate keys according to the language request headers
     res.render('index', {
-        title: req.translate('title')
+        title: req.translate('title')        
+        message: req.translate('greeting');
     });
 });
 
 app.listen(3000);
+...
 ```
+## Configuration Options
+
+| Option              | Description |
+|---------------------|-------------|
+| `baseDir`           | Root directory for resource lookup based on path. Default: process.cwd(). 
+| `allowedLangs`      | List of allowed languages 
+| `locale`            | Default locale
+| `cookie`            | Cookie settings (e.g., `{ name: 'lang' }`) 
+| `query`             | Name of query parameter 
+| `noCache`           | Disable caching of resources 
+| `watch`             | Watch for resource file changes 
+| `setHeaders`        | Set Content-Language header 
+| `fallbackLang`      | Sets the fallback language if none is specified or a key is missing. Default: 'en'.
```

Comment:
Good to show some usage of the fallback options? Or have an example using it and link to it from here?